sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -38,82 +37,147 @@
 
 
 class SdkHarness(object):
+  REQUEST_METHOD_PREFIX = '_request_'
+
+  def __init__(self, control_address, pipeline_options=None):
+    def _get_worker_count(pipeline_options):
+      pipeline_options = pipeline_options.get(
+          'options'
+      ) if pipeline_options and pipeline_options.has_key('options') else {}
+      experiments = pipeline_options.get(
+          'experiments'
+      ) if pipeline_options and pipeline_options.has_key('experiments') else []
+
+      experiments = experiments if experiments else []
+
+      for experiment in experiments:
+        # There should only be 1 match so returning from the loop
+        s = re.findall('^(worker_threads=[0-9]+)', experiment)
+        if len(s) > 0:
+          return int(s.pop()[len('worker_threads='):])
```

Comment:
A little nit. Is there a simpler (easier to read) way to express this statement?